src/crypto/crypto_keygen.h

```diff
@@ -200,8 +200,8 @@ struct KeyPairGenTraits final {
 };
 
 struct SecretKeyGenConfig final : public MemoryRetainer {
-  size_t length;  // Expressed a a number of bits
-  char* out = nullptr;  // Placeholder for the generated key bytes
+  size_t length;  // Expressed as a number of bytes.
```

Comment:
Since we're talking nits. :p
```suggestion
  size_t length;  // Expressed as the number of bytes.
```
(although I'd just write `// In bytes.`)